app/src/main/java/fr/free/nrw/commons/upload/UploadActivity.java

```diff
@@ -700,6 +746,23 @@ protected void onDestroy() {
             uploadCategoriesFragment.setCallback(null);
         }
     }
+    /**
+     * Get the value of the showPermissionDialog variable.
+     *
+     * @return {@code true} if Permission Dialog should be shown, {@code false} otherwise.
+     */
+    public boolean isShowPermissionsDialog() {
+        return showPermissionsDialog;
+    }
+    /**
```

Comment:
1 empty line between each method.

Reply:
Sure, I will try to complete all requested changes